extra-dev/packages/coq-flocq/coq-flocq.dev/opam

```diff
@@ -1,12 +1,15 @@
 opam-version: "2.0"
-maintainer: "guillaume.melquiond@inria.fr"
+maintainer: "everybody"
 homepage: "http://flocq.gforge.inria.fr/"
 dev-repo: "git+https://scm.gforge.inria.fr/anonscm/git/flocq/flocq.git"
 license: "LGPL 3"
 build: [
   ["./autogen.sh"]
   ["./configure"]
-  ["./remake" "-j%{jobs}%"]
+  ["find" "src" "-name" "*.v" "-fprint" "vfiles"]
+  ["sh" "-c" "cat _CoqProject vfiles > _CoqProject.full"] # how else to add to a file in opam?
```

Comment:
https://opam.ocaml.org/doc/Manual.html#opam
Look for `patches:`, `extra-files:` and `extra-sources:`.

Your solution is also ok. You can also use `coq_makefile -o Makefile -f _CoqProject more files here`.